tests/functional/centos/7/mon-osd/hosts

```diff
@@ -0,0 +1,5 @@
+[mons]
+mon0 monitor_interface=enp0s8
```

Comment:
excellent. Do you know if it is possible to add comments here? It would be nice to say why `enp0s8` is needed